packages/compiler-core/src/transforms/transformElement.ts

```diff
@@ -54,10 +57,16 @@ export const transformElement: NodeTransform = (node, context) => {
         if (isComponent) {
           context.helper(RESOLVE_COMPONENT)
           context.components.add(node.tag)
+        } else if (isPortal) {
+          context.helper(PORTAL)
```

Comment:
This branch is unnecessary because `context.helper(PORTAL)` will be called below for the first argument anyway.